hasher-matcher-actioner/hmalib/lambdas/api/lcc.py

```diff
@@ -0,0 +1,65 @@
+from dataclasses import dataclass, asdict
+from mypy_boto3_dynamodb.service_resource import Table
+import bottle
+
+from hmalib.common.config import HMAConfig
+from hmalib.common.models.bank import BankMember, BanksTable
+
+from hmalib.lambdas.api.middleware import (
+    jsoninator,
+    JSONifiable,
+    DictParseable,
+    SubApp,
+)
+
+import typing as t
+
+
+# I return this type because I saw in the documentation that a
+# to_json method is required
+
+# I might need to use
+# `jsoninator(<requestObj>, from_query=True)` but not sure how the from_query=True
```

Comment:
From query is useful if your API _request_ payload has a type. In this case, that's not true. Using `bottle.request.query` is sufficient. You'll only need a single argument to the jsoninator decorator.